src/core/Akka/Actor/ActorSelection.cs

```diff
@@ -348,7 +348,7 @@ public SelectChildPattern(string patternStr)
         /// Returns a <see cref="string" /> that represents this instance.
         /// </summary>
         /// <returns>A <see cref="string" /> that represents this instance.</returns>
-        public override string ToString() => PatternStr.ToString(CultureInfo.InvariantCulture);
+        public override string ToString() => PatternStr.ToString();
```

Comment:
This overloaded method is not supported in netcore, and don't have any sense in .NET 45
http://referencesource.microsoft.com/#mscorlib/system/string.cs,2648